components/VM_Arm/src/main.c

```diff
@@ -715,6 +717,34 @@ static int route_irqs(vm_vcpu_t *vcpu, irq_server_t *irq_server)
     return 0;
 }
 
+/* Different from load_image which loads the image into the guest's address space. This pulls the
+ * image from the FileServer and stores it into a local buffer
+ */
+static int get_file(const char *name, char *buf)
```

Comment:
This returns a length or 0 and not an error code, so the return type should be `size_t` then.
```suggestion
static size_t get_file(const char *name, char *buf)
```

Reply:
Good call. I've updated the return value.